packages/webview_flutter/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: webview_flutter_tizen
 description: Tizen implementation of the webview plugin
 homepage: https://github.com/flutter-tizen/plugins
-version: 0.2.2
+version: 0.2.3
```

Comment:
Since this is a breaking change (the engine update is required), you might want to increase the version to 0.3.0.

Reply:
Ok thanks :smile: